code/game/objects/items/manuals.dm

```diff
@@ -168,8 +168,12 @@
 				Use it to cook or boil food ingredients (meats, doughs, egg, spaghetti, donkpocket, etc...).
 				It can cook multiple items at once.
 
+<!-- honk start -- contains gamer words
```

Comment:
I never thought about this case. But it makes sense, I'll have to update my tool. That being said, we don't need two copies, just change the relevant section